README.md

```diff
@@ -162,42 +162,42 @@ utPLSQL 3.1.2.1913
 ```
 > utplsql reporters app/app@localhost:1521/ORCLPDB1
 
-UT_COVERAGE_COBERTURA_REPORTER (SQL): Generates a Cobertura coverage report providing information on code coverage with line numbers.
-Designed for Jenkins and TFS to report coverage. 
-Cobertura Document Type Definition can be found: http://cobertura.sourceforge.net/xml/coverage-04.dtd.
-Sample file: https://github.com/leobalter/testing-examples/blob/master/solutions/3/report/cobertura-coverage.xml.
+UT_COVERAGE_COBERTURA_REPORTER Generates a Cobertura coverage report providing information on code coverage with line numbers.
```

Comment:
With my pedantic hat on - wouldn't it be better to have it formatted with indentation of 4 spaces?
```
UT_COVERAGE_COBERTURA_REPORTER:
    Generates a Cobertura coverage report providing information on code coverage with line numbers.
    Designed for Jenkins and TFS to report coverage.
```
Indenting by up to 31 chars seems excessive